app/_layouts/base.njk

```diff
@@ -15,7 +17,9 @@
 {% endblock %}
 
 {% block pageTitle %}
-  {% if title %}{{ title }} - {% endif %}{{ app.serviceName or app.productName }} - GOV.UK
+  {{- title if title -}}
+  {{- " (page " + pageNumber + " of " + pageCount + ")" if pageCount > 1 -}}
```

Comment:
```suggestion
  {{- " (page " + pageNumber + " of " + pageCount + ")" if (pageCount > 1 and pageNumber > 1) -}}
```

Reply:
I think the pagination title should only show on subsequent pages, ie not the first – what do you think?

Reply:
Makes sense to me!